action.yml

```diff
@@ -41,6 +41,10 @@ inputs:
     description: 'deploy/promote/reject'
     required: true
     default: 'deploy'
+  force:
+    description: 'Deploy when a previous deployment already exists'

```

Comment:
Add something like.. `adds `--force` argument to the apply command` or something on those lines?

Reply:
added